lib/js/jslib.js

```diff
@@ -1761,31 +1524,26 @@ function _swapNodes(x, y) {
   if(yNextSibling != x)
     yParent.insertBefore(x, yNextSibling);
 
-  return {}
+  return CONSTANTS.UNIT;
 }
 var swapNodes = LINKS.kify(_swapNodes);
 
-//replaceChildren : xml -> domRef -> ()
-function _replaceChildren(xml, parent) {
+/**
+ * domReplaceChildren : xml -> domRef -> ()
+ * @param {Xml} xml
+ * @param {DomNode} parent
+ * @returns {()}
+ */
+function _domReplaceChildren(xml, parent) {
   newNodes = LINKS.XmlToDomNodes(xml);
```

Comment:
Sure, you're right, my comment was over-specific on the type of binder needed.

Not catching them all straight away is OK -- but ideally we want the examples to work.